api/registry.py

```diff
@@ -125,6 +126,10 @@ class Registration(object):
     PRODUCTION_STAGE = "production"
     VALID_REGISTRATION_STAGES = [TESTING_STAGE, PRODUCTION_STAGE]
 
+    # A registry may provide access to a web client. If so, we'll store
+    # the URL so we can link to it.
```

Comment:
I was also confused by this until you explained that we'll "link to it" by generating MARC records. Mentioning that here would be really helpful.